securedrop/source_templates/tor2web-warning.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% block body %}
 <h1>{{ gettext('Why is there a warning about Tor2Web?') }}</h1>
-<p>{{ gettext('<a href="tor2web.org">Tor2Web</a> is a proxy service that lets you browse Tor Hidden Services (.onion sites) without installing Tor. It was designed to facilitate anonymous publishing.') }}</p>
-<p>{{ gettext('Tor2Web only protects publishers, not readers. If you upload documents to us using Tor2Web, you are <strong>not anonymous</strong> and could be identified by your ISP or the Tor2Web proxy operators. Additionally, since Tor2Web sites typically do not use HTTPS, it is possible that your connection could be MITM\'ed by a capable adversary.') }}</p>
-<p>{{ gettext('If you want to submit information, you are <strong>strongly advised</strong> to install <a href="torproject.org/download.html.en">Tor</a> and use it to access our site safely and anonymously.') }}</p>
+<p>{{ gettext('Using SecureDrop with <a href="tor2web.org">Tor2Web</a> will not protect your anonymity.') }}</p>
+<p>{{ gettext('It could be possible for anyone listening to Internet usage (your government, your Internet provider), to identify you.') }}</p>
+<p>{{ gettext('If you want to submit information to us we <strong>strongly advised</strong> installing the <a href="www.torproject.org/projects/torbrowser.html">Tor browser</a> and upload documents using that.') }}</p>
```

Comment:
Current: 

> 'If you want to submit information to us we <strong>strongly advised</strong> installing the <a href="www.torproject.org/projects/torbrowser.html">Tor browser</a> and upload documents using that.'

Should be

> 'If you want to submit information to us we <strong>strongly advised</strong> installing the <a href="www.torproject.org/projects/torbrowser.html">Tor browser</a> using it to visit SecureDrop.'

We should always say files instead of documents, and we also want to keep the messaging consistent in that we accept files and messages alike. We could replace "using it to visit SecureDrop" with "using it to send us files and messages," but that would imply that visiting SD without TB is safe when it is not.

Reply:
@ei8fdb I would like your input on this since I might be a bit off with my analysis of the exact phrasing here.

Reply:
I suggest we just remove **and upload documents using that** entirely. It is implied since the sentence starts with **If you want to submit information to us**. Leaving it sounds like: **If you want to send a us message, use the phone to send us a message** ;-)

Reply:
Event better.

Reply:
What about **If you want to submit information to us we <strong>strongly advise</strong> you to use the...** ?

Reply:
Or even **We strong advise you to use the tor browser instead.**